src/services/services.ts

```diff
@@ -1171,6 +1171,26 @@ namespace ts {
         }
     }
 
+    const invalidOperationsOnSyntaxOnly: readonly (keyof LanguageService)[] = [

```

Comment:
How does this relate to the list in session.ts?  Do they need to stay in sync?  Is one redundant?